modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -22,48 +34,76 @@ proxy:
+* Deploy the `red-hat-developer-hub-customization-provider` service and the {product-short} Helm chart on the same cluster.
 
 .Prerequisites
 
-* You have installed the {product} using Helm Chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+* You have installed the {product} using the Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
 
 .Procedure
 
-. In Red Hat OpenShift, select *+Add* and click *Import from Git* option.
-. Add the URL of your Git repository to the *Git Repo URL* field.
+To use a separate service to provide the Home page data, complete the following steps:
+
+. From the *Developer* perspective in the {ocp-brand-name} web console, click *+Add* > *Import from Git*.
+. Enter the URL of your Git repository into the *Git Repo URL* field.
 +
 --
-To use the `red-hat-developer-hub-customization-provider` service, you can add the URL of https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository.
+To use the `red-hat-developer-hub-customization-provider` service, add the URL for the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository or your fork of the repository containing your customizations.
 --
 
-. In the *General* section, rename the value in the *Name* field to *rhdh-customization-provider* and click *Create*.
-. Go to the *Advanced Options* and copy the value from the *Target Port*.
+. On the *General* tab, enter *red-hat-developer-hub-customization-provider* in the *Name* field and click *Create*.
+. On the *Advanced Options* tab, copy the value from the *Target Port*.
 +
---
-The *Target Port* is used to automatically generate a Kubernetes or OpenShift service to communicate with.
---
+[NOTE]
+====
+The *Target Port* automatically generates a Kubernetes or {ocp-short} service to communicate with.
+====
++
+. Add the following code to the `app-config-rhdh.yaml` file:
++
+[source,yaml]
+----
+proxy:
+  endpoints:
+    # Other Proxies
+    # customize developer hub instance
+    '/developer-hub':
+      target: ${HOMEPAGE_DATA_URL}
+      changeOrigin: true
+      # Change to "false" in case of using self-hosted cluster with a self-signed certificate
+      secure: true
+----
+where `HOMEPAGE_DATA_URL` is defined as `pass:c[http://<SERVICE_NAME>:8080]`, for example, `pass:c[http://rhdh-customization-provider:8080]`.
++
+[NOTE]
+====
+The `red-hat-developer-hub-customization-provider` service contains the 8080 port by default. If you are using a custom port, you can specify it with the 'PORT' environmental variable in the `app-config-rhdh.yaml` file.
+====
++
+[NOTE]
+====
+You can replace the `HOMEPAGE_DATA_URL` by adding the URL to `rhdh-secrets` or by directly replacing it in your custom ConfigMap.
```

Comment:
@Zaperex is this an optional step or must the user replace the `HOMEPAGE_DATA_URL` by using one of these two methods?

Reply:
This is how we configure the url the proxy queries for the home page information so this is not optional. They could either provide use `HOMEPAGE_DATA_URL` in the `app-config` and then set it's value in a `Secret` or directly provide it in the `app-config` config map

Reply:
Will convert this admonition into a required step in the procedure